ColorPatterns/Scenes/Game/GameViewModel.swift

```diff
@@ -36,25 +48,26 @@ extension GameViewModel {
     }
     
     func didPressColorBar(colorBar: Int) {
+        gameSounds.play(soundFile: colorBar)
         gameEngine.pickedColor = gameEngine.colorsArray[colorBar]
         calculateScore()
         gameStarted = true
         helpPrints()
     }
 }
 
-private extension GameViewModel {
+private extension GameViewModelImpl {
 
     private func calculateScore() {
         if gameStarted {
             userScore.updateScorePoints(gameEngine.colorRange.contains(gameEngine.pickedColor))
-            delegate?.didUpdateScore(score: userScore.score)
+            controller?.updateScoreLabel(score: userScore.score)
         } else {
             gameEngine.userColor = gameEngine.pickedColor
         }
     }
 
-    //TODO: Remove prints when not needed
+    // TODO: Remove prints when not needed
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/b646116fac92a792fe76a76051e573c13372835d/ColorPatterns/Scenes/Game/GameViewModel.swift#L70"></span>TODOs should be resolved (Remove prints when not needed).<br />
<code>todo</code> <code>GameViewModel.swift:70</code></td>
    </tr>
  </tbody>
</table>